app/controllers/application_controller.rb

```diff
@@ -321,10 +321,10 @@ def confirm_two_factor_authenticated(id = nil)
     return prompt_to_setup_mfa unless two_factor_enabled?
     return prompt_to_verify_mfa unless user_fully_authenticated?
     return prompt_to_setup_mfa if service_provider_mfa_policy.
-                                  user_needs_sp_auth_method_setup?
+      user_needs_sp_auth_method_setup?
```

Comment:
this one is weird. I think we should use the block syntax in cases like this?
```
if service_provider_mfa_policy. user_needs_sp_auth_method_setup?
  return prompt_to_setup_mfa
end
```

Reply:
The problem I see here is that there's a lot of them. I'd probably convert the whole block:

```ruby
    return prompt_to_setup_mfa unless two_factor_enabled?
    return prompt_to_verify_mfa unless user_fully_authenticated?
    return prompt_to_setup_mfa if service_provider_mfa_policy.
      user_needs_sp_auth_method_setup?
    return prompt_to_setup_non_restricted_mfa if two_factor_kantara_enabled?
    return prompt_to_verify_sp_required_mfa if service_provider_mfa_policy.
      user_needs_sp_auth_method_verification?
```

Mixing them looks weird otherwise.

Incidentally, this one line doesn't actually have to be broken to not be too long, but the one a few lines down does.

Are you on board with me converting this whole section? It'll be a bit longer but I think it's for the best.

Reply:
I don't mind mixing, but I do think that it would be preferable to have the blocks to the split trailing ones like originally proposed